server/src/schemas/holdsSchema.ts

```diff
@@ -1,38 +1,26 @@
 import { gql } from "apollo-server-express";
 
-export interface Hold {
-  id: number;
-  description: String;
-}
-
 export const HoldsTypeDefs = gql`
-  enum HoldType {
-    INCOMPLETE_TRAINING
-    SAFETY_VIOLATION
-    BALANCE_DUE
-  }
-
   type Hold {
     id: ID!
-    type: HoldType!
+    userID: User!
     description: String
-    createdBy: User!
-    createdAt: DateTime
-    removedAt: DateTime
-    userId: User!
+    active: Boolean
   }
 
   input HoldInput {
-    id: Int!
-    type: HoldType!
+    userID: Int!
     description: String
-    createdBy: Int!
-    createdAt: DateTime
-    removedAt: DateTime
-    userId: Int!
+    active: Boolean
   }
 
   type Query {
     holds: [Hold]
+    holdsByUser(userID: ID!): [Hold]
```

Comment:
IMO we can get rid of both of these queries, since we can just query `user.holds` instead.

Reply:
I don't think `user.holds` which returns an array should exist. This is actually like a lookup table for holds which is better imo. @MatthewGalan I think we should segregate holds functionality from the Users stuff too

Reply:
I added the `userID` field so that we no longer need to keep track of holds in `Users`

Reply:
IMO the API design should be independent from the database design. Conceptually speaking, holds are attached to users, and thus should be accessed via `user.holds`. `holdsByUser` feels like something from a REST API, not a GraphQL API.

CC: @pjw9963 thoughts?

Reply:
I think a hold is always connected to a user, making it like a property. Graphql connects our properties like a graph, so the end-user can query for a user object and select whatever properties they want as long as they are related. I think separating the logic and db implementation of users and holds is good but the graphql resolver should tie them together and allow for the end-user to query for a user and their corresponding holds.